### PR TITLE
[MINOR] Improving transaction feature.

### LINKED ISSUE #199: Transaction feature modification request
- Instead of “view production” write “money received”
- Instead of “quantity” transform to “payment type”:
progress, final payment, deposit
- Include the name of the client -> should put it in the information
- Include the name of the employee working on the project -> put it as a scrollable 
- Instead of “net profit” put “gross profit”

### PR DESCRIPTION
**Pull request by Mnx458**
closes #199

image

### CODE REVIEW

File: backend/src/main/dto/InvoiceDTO.ts [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 export interface InvoiceUpdateDTO {
   totalAmount?: number;
-  quantity?: number;
+  paymentType?: paymentType;
   date?: Date;
   description?: string;
 }
+
+export enum paymentType {
+  PROGRESS = 'PROGRESS',
```
Comment (huangs08) [inline backend/src/main/dto/InvoiceDTO.ts:16]: I believe IN_PROGRESS would be less confusing than PROGRESS. Thoughts?

File: backend/src/main/models/Invoice.ts [hunk 2]
```diff
@@ -11,8 +21,9 @@ export class Invoice extends Model {
   @Column
   totalAmount!: number;
 
-  @Column
-  quantity!: number;
+  @Default('PROGRESS')
+  @Column(DataType.ENUM('PROGRESS', 'FINAL PAYMENT', 'DEPOSIT'))
```
Comment (huangs08) [inline backend/src/main/models/Invoice.ts:25]: Same comment about PROGRESS here.

File: backend/src/main/dto/InvoiceDTO.ts (hunk 1 above)
Comment (Mnx458) [reply backend/src/main/dto/InvoiceDTO.ts:16]: Stakeholders told us to put "progress" according to the meeting notes.

Comment (Manraj45) [reply backend/src/main/dto/InvoiceDTO.ts:16]: Yeah I think "in progress" would be better, and I think "deposit" should be "initial deposit" but ask the stakeholders before making the change.

Comment (Mnx458) [reply backend/src/main/dto/InvoiceDTO.ts:16]: I think we should follow the stakeholders meeting notes for now and ask them later.

**Review by Manraj45 — APPROVED** [review]
Good job!

**Review by huangs08 — APPROVED** [review]

**Pull request merged**